.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: flutter packages pub run build_runner build --delete-conflicting-outputs
 
       - name: Check for any formatting issues in the code
-        run: flutter format --set-exit-if-changed .
+        run: dart format --set-exit-if-changed .
```

Comment:
FYI: This command upgrade is required. However, upgrading to use `dart run` is not required yet s,o we will do it later to not keep too much things in this PR 🙏